templates/confluence.cfg.xml.erb

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<confluence-configuration>
+  <setupStep>setupdbchoice-start</setupStep>
+  <setupType>custom</setupType>
+  <buildNumber>0</buildNumber>
+  <properties>
+    <property name="atlassian.license.message"><%= scope.lookupvar('confluence::license') -%></property>
+    <property name="confluence.setup.server.id"><%= scope.lookupvar('confluence::server_id') -%></property>
+    <property name="confluence.webapp.context.path"></property>
+    <property name="lucene.index.dir">${localHome}/index</property>
+    <property name="synchrony.btf">true</property>
+    <property name="synchrony.encryption.disabled">true</property>
+    <property name="synchrony.proxy.enabled">true</property>
+    <property name="webwork.multipart.saveDir">${localHome}/temp</property>
+  </properties>
+</confluence-configuration>
```

Comment:
Yes, the static values are the defaults. Confluence has about 5 "startup screens" in the browser which allow the admin to paste in the license key and some other values. Eventually I would like to get to full setup automation (why else use Puppet?) but the steps are non-obvious beyond the choose database step. This at least shortens setup a little.